airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -2245,9 +2245,10 @@ class CustomObject(CRMSearchStream, ABC):
     primary_key = "id"
     scopes = {"crm.schemas.custom.read", "crm.objects.custom.read"}
 
-    def __init__(self, entity: str, schema: Mapping[str, Any], fully_qualified_name: str, custom_properties: Mapping[str, Any], **kwargs):
+    def __init__(self, entity: str, name: str, schema: Mapping[str, Any], fully_qualified_name: str, custom_properties: Mapping[str, Any], **kwargs):
         super().__init__(**kwargs)
         self.entity = entity
+        self.name = name
```

Comment:
we already have getter for `name` attribute, so better to proceed with this logic and add setter

Reply:
I modified the name getter logic instead - this is much cleaner. Let me know what you think.